# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bglibs/no-selftests.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- bglibs-1.101/Makefile.vanilla	2006-10-04 15:37:41.000000000 +0000
+++ bglibs-1.101/Makefile	2006-10-04 15:37:52.000000000 +0000
@@ -126,7 +126,7 @@
 adt/hashs.o: compile adt/hashs.c adt/common.h
 	./compile adt/hashs.c
 
-all: dotlibs sysdeps.h libraries bg-installer cli-generate selftests
+all: dotlibs sysdeps.h libraries bg-installer cli-generate
 
 base64/asc2bin.lo: libcompile base64/asc2bin.c base64/base64.h str/str.h
 	./libcompile base64/asc2bin.c
